libguard/guard_interface.cpp

```diff
@@ -124,6 +124,25 @@ static GuardRecord getHostEndiannessRecord(const GuardRecord& record)
     return convertedRecord;
 }
 
+/**
+ * @brief Function to get the size remaining in the file
+ *
+ * @param[in] size - File size
+ * @param[in] count - No of records present
```

Comment:
No of records present in the guard file?

Reply:
Yes, will change it.

Reply:
Done.